modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Tags/ITagManager.cs

```diff
@@ -26,5 +27,7 @@ public interface ITagManager : IDomainService
             [NotNull] string name,
             Guid? tenantId = null,
             CancellationToken cancellationToken = default);
+
+        Task<List<TagDefiniton>> GetTagDefinitionsAsync(CancellationToken cancellationToken = default);
```

Comment:
No need to define this method as `async`.

Reply:
https://github.com/abpframework/abp/pull/7103#discussion_r555686131